packages/mdc-list/README.md

```diff
@@ -71,8 +71,8 @@ While in theory you can add any number of "lines" to a list item, you can use th
 <ul class="mdc-list mdc-list--two-line">
   <li class="mdc-list-item">
     <span class="mdc-list-item__text">
-      <span class="mdc-list-item__text__primary">Two-line item</span>
-      <span class="mdc-list-item__text__primary">Secondary text</span>
+      <span class="mdc-list-item__text">Two-line item</span>
```

Comment:
Remove the `<span>` elements wrapping `Two-line item` here and below (this matches what's in the demo)